apps/studio/components/interfaces/SignIn/SignInForm.tsx

```diff
@@ -106,76 +116,78 @@ const SignInForm = () => {
-                onVerify={(token) => {
-                  setCaptchaToken(token)
-                }}
-                onExpire={() => {
-                  setCaptchaToken(null)
-                }}
-              />
-            </div>
-
-            <LastSignInWrapper type="email">
-              <Button
-                block
-                form="signIn-form"
-                htmlType="submit"
-                size="large"
-                disabled={isSubmitting}
-                loading={isSubmitting}
-              >
-                Sign In
-              </Button>
-            </LastSignInWrapper>
-          </div>
-        )
-      }}
-    </Form>
+    <Form_Shadcn_ {...form}>
+      <form id={formId} className="flex flex-col gap-4" onSubmit={form.handleSubmit(onSubmit)}>
+        <FormField_Shadcn_
+          key="email"
+          name="email"
+          control={form.control}
+          render={({ field }) => (
+            <FormItemLayout name="email" label="Email">
+              <FormControl_Shadcn_>
+                <Input_Shadcn_
+                  id="email"
+                  type="email"
+                  autoComplete="email"
+                  {...field}
+                  placeholder="you@example.com"
+                />
+              </FormControl_Shadcn_>
+            </FormItemLayout>
+          )}
+        />
+
+        <div className="relative">
+          <FormField_Shadcn_
+            key="password"
+            name="password"
+            control={form.control}
+            render={({ field }) => (
+              <FormItemLayout name="password" label="Password">
+                <FormControl_Shadcn_>
+                  <Input_Shadcn_
+                    id="password"
+                    type="password"
+                    autoComplete="current-password"
+                    {...field}
+                    placeholder="&bull;&bull;&bull;&bull;&bull;&bull;&bull;&bull;"
+                    disabled={isSubmitting}
+                  />
+                </FormControl_Shadcn_>
+              </FormItemLayout>
+            )}
+          />
+
+          {/* positioned using absolute instead of labelOptional prop so tabbing between inputs works smoothly */}
+          <Link
+            href={forgotPasswordUrl}
+            className="absolute top-0 right-0 text-sm text-foreground-lighter"
+          >
+            Forgot Password?
+          </Link>
+        </div>
+
+        <div className="self-center">
+          <HCaptcha
+            ref={captchaRef}
+            sitekey={process.env.NEXT_PUBLIC_HCAPTCHA_SITE_KEY!}
+            size="invisible"
+            onVerify={(token) => {
+              setCaptchaToken(token)
+            }}
+            onExpire={() => {
+              setCaptchaToken(null)
+            }}
+          />
+        </div>
+
+        <LastSignInWrapper type="email">
+          <Button block form={formId} htmlType="submit" size="large" loading={isSubmitting}>
```

Comment:
note that `loading` will implicitly cover `disabled`, so there is no need to provide both if they are the same condition